Mathlib/CategoryTheory/Monoidal/Bimon_.lean

```diff
@@ -68,7 +68,6 @@ def toComon_ : Bimon_ C ⥤ Comon_ C := (Mon_.forgetMonoidal C).toOplaxMonoidalF
 @[simp]
 theorem toComon_forget : toComon_ C ⋙ Comon_.forget C = forget C := rfl
 
-set_option maxHeartbeats 400000 in
```

Comment:
Let's just squeeze the simp. I think it will trip the alarm again soon.

Reply:
Well, this is what happens:
```lean
-- Used 166713 heartbeats
count_heartbeats in
/-- The object level part of the forward direction of `Comon_ (Mon_ C) ≌ Mon_ (Comon_ C)` -/
def toMon_Comon_obj (M : Bimon_ C) : Mon_ (Comon_ C) where
  X := (toComon_ C).obj M
  one := { hom := M.X.one }
  mul :=
  { hom := M.X.mul,
    hom_comul := by dsimp only [Monoidal.transportStruct_tensorObj, Equivalence.symm_functor,
      Comon_.Comon_EquivMon_OpOp_inverse, Equivalence.symm_inverse,
      Comon_.Comon_EquivMon_OpOp_functor, Comon_.Comon_ToMon_OpOp_obj, Comon_.Mon_OpOpToComon__obj,
      unop_tensorObj, Comon_.Mon_OpOpToComon_obj'_X, Mon_.monMonoidalStruct_tensorObj_X,
      Comon_.Comon_ToMon_OpOp_obj'_X, toComon__obj_X, Comon_.Mon_OpOpToComon_obj'_comul,
      Mon_.tensorObj_mul, Comon_.Comon_ToMon_OpOp_obj'_mul, unop_comp, unop_tensorHom,
      Quiver.Hom.unop_op]; simp only [toComon__obj_comul, Mon_.Hom.mul_hom,
      Mon_.monMonoidalStruct_tensorObj_X, Mon_.tensorObj_mul, tensor_μ, Category.assoc, unop_comp,
      unop_tensorObj, unop_inv_associator, unop_whiskerLeft, unop_hom_associator, unop_whiskerRight,
      unop_hom_braiding] }
```
So, squeezing does improve the speed, but I am not really sure that it is such a great improvement!

Maybe we can live with the `set_option`...

Reply:
Yes, I'd prefer the `set_option` here, hopefully provoking a better fix in future.

Reply:
Aside: using `count_heartbeats` and `simp?` explicitly makes lean run forever on my machine despite the file appearing finished